lib/model/store.dart

```diff
@@ -287,6 +287,12 @@ class PerAccountStore extends ChangeNotifier with StreamStore {
 
   final Map<int, User> users;
 
+  Map<int, Set<int>> get streamUserIds => {..._streamUserIds};
```

Comment:
This getter copies the whole Map. That's an O(N) operation (in the number of streams found in the map), so it shouldn't be a getter. See discussion here:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#getters-feel-faster-than-methods

In this case, though, the caller only needs a single entry from the map. So we can instead make it a method like (untested) `Iterable<int> knownSendersInStream(int streamId) => _knownSendersInStream[streamId];`